cdap-app-fabric/src/main/java/org/apache/twill/yarn/YarnTwillRunnerService.java

```diff
@@ -329,20 +330,6 @@ private void startUp() throws Exception {
 
     watchCancellable = watchLiveApps();
     liveInfos = createLiveInfos();
-
-    // Schedule an updater for updating HDFS delegation tokens
-    if (UserGroupInformation.isSecurityEnabled()) {
```

Comment:
Since cdap has scheduled its own SecureStoreUpdater, we don't need to have the one in Twill also performing updates.
I filed a relevant JIRA as I found this: https://issues.apache.org/jira/browse/TWILL-185